app/models/post.rb

```diff
@@ -1,2 +1,13 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
```

Comment:
[REQUIRED] Kindly remove foreign_key has_many :comments and has_many :likes. It is meant to be used when you are trying to reference another model with another name

Reply:
Done